Source/Stationeers.Addons/Core/LoaderManager.cs

```diff
@@ -143,6 +143,25 @@ private IEnumerator Start()
             }
         }
 
+        private void Update()
+        {
+            if (Input.GetKeyDown(KeyCode.R) && Input.GetKey(KeyCode.LeftControl))
+            {
+                Debug.Log("REC update");
+                PluginLoader.UnloadAllPlugins();
+                StartCoroutine(Reload());
```

Comment:
To avoid some isses when user presses Ctrl+R multiple times, it would be a good idea to store the coroutine/or some state/flag in a variable, and disallow code reload until the previous one finishes.

Reply:
Definitely. I was thinking about adding a label during the recompilation, one of the monobehaviours already had an OnGui() method. It will also take a try/catch to ensure we get out of the compiling state

Reply:
forgot you can' t `try/catch` a `yield return`. well, I reasonably shielded all methods involved instead. while recompiling, the label will display it instead of the usual "n plugins loaded"